api/inviteUser/inviteUserUtil.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { expect } from 'chai'
+import { getInviteUserUrl } from './inviteUserUtil'
+
+describe('terms And Conditions ', () => {
```

Comment:
what does this have to do with T&C's? lol

Reply:
lol. fixed it now.